### PR TITLE
Fixes #66 now pressing on active button doesn't toggle the class

### LINKED ISSUE #66: pressing the active format button should not make it inactive
when an active format button is clicked, it makes it inactive and sets format=None in the request parameters. this is leading to an error 400 error.
**before pressing the button**
image
**after pressing the button**
image

### PR DESCRIPTION
**Pull request by nikhilrayaprolu**
Fixes #66 removed the line that makes the button inactive.
@enigmaeth @mariobehling review.

### CODE REVIEW

**Comment by enigmaeth** [comment]
Please fix typo in the commit message :smile: . Doesn't look good!

**Comment by nikhilrayaprolu** [comment]
@enigmaeth changed the commit message :smile:

File: app/templates/index.html [hunk 1]
```diff
@@ -111,7 +111,7 @@
     $('.formatButton').click(function(e){
         e.preventDefault();
         if($(this).hasClass('active')){
-            $(this).removeClass('active')
+           
         } else {
```
Comment (rhemon) [inline app/templates/index.html]: now that `if` block is unnecessary, it should be removed. you could instead now directly check if it doesn't have the class `active`, and then run the block of code inside `else`

**Comment by rhemon** [comment]
your branch is 1 commit behind, a rebase and it should be ready for merge

**Comment by nikhilrayaprolu** [comment]
@rhemon done.

**Comment by rhemon** [comment]
cool. one more thing, i dont think its good practice to have so manny commits in one simple pr where change serves only one purpose. usually comits should be for each meaningful change. some good github tutorial links are available, incase you dont know of squashing, rebase and amend, [here](https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/CONTRIBUTING.md) . try to keep those practices in your prs :)

**Comment by enigmaeth** [comment]
Squash the commits and rebase it on top of master. Rest looks good.

**Comment by enigmaeth** [comment]
Squash and rebase please^^

**Pull request merged**